### DISCUSSION TITLE
subdir/about/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#19] [General]
# subdir/about/

Hugo, the world's fastest framework for building websites

http://localhost:1313/subdir/about/

**Comment by blanche789** [comment]
hello

**Comment by blanche789** [comment]
yes

**Comment by peng-du-cd** [comment]
123213

**Comment by Paul-PINTEAU** [comment]
J'ai hâte de suivre les aventures de Monsieur Fortran à travers le désert américain !

**Comment by smilingott** [comment]
test

**Comment by holbrocr** [comment]
testing

**Reply by holbrocr** [reply]
my reply

**Comment by antoniojorge27** [comment]
hu

**Comment by txxnadmin** [comment]
Test comment by Andrew Hill

**Comment by tochiko-0** [comment]
test
